cookbooks/bcpc/recipes/powerdns.rb

```diff
@@ -45,259 +44,214 @@
 subnet = node[:bcpc][:management][:subnet]
 raise "Did not get a subnet" if not subnet
 
-%w{pdns-server pdns-backend-mysql}.each do |pkg|
-    package pkg do
-        action :upgrade
-    end
-end
+if node[:bcpc][:management][:vip] and get_nodes_for("mysql").length() > 0
+#  make_config('mysql-pdns-user', "pdns")
+#  make_config('mysql-pdns-password', secure_password)
+
+  allnodes = Array.new
```

Comment:
This is just a local variable the following [code](https://github.com/amithkanand/chef-bach/blob/f1c3a54f072a1c88fc27e53b1b194fd8ab05fcf8/cookbooks/bcpc/recipes/powerdns.rb#L51-L52] uses `get_all_nodes` and populates this local variable.